### PR TITLE
Updated text

### PR DESCRIPTION
**Pull request by philippberens**
I started updating the text for the about and the download page. Please review.

### CODE REVIEW

File: client/components/panels/submit.js [hunk 1]
```diff
@@ -126,13 +126,31 @@ function submit (state) {
...
   <div style=${style.form}>
     <div>
     <span style=${style.label}>team name</span> <input id='name' style=${style.input}>
     </div>
+	<div>
+    <span style=${style.label}>member names</span> <input id='members' style=${style.input}>
+    </div>
+	
+	<div>
+    <span style=${style.label}>affiliation(s)</span> <input id='affiliation' style=${style.input}>
```
Comment (freeman-lab) [inline client/components/panels/submit.js]: I'm all for adding real names of the submitters, but affiliation might not make sense for everyone that we'd like to take part -- e.g. some coder not part of a university or company, so maybe drop that one?

**Comment by freeman-lab** [comment]
Cool, just left one comment, otherwise looks good to merge!

**Comment by philippberens** [comment]
Alright, I removed the field. Have a look. We have to see whether we really want all the logos on the starting page.

**Comment by freeman-lab** [comment]
Cool let's merge this in, as nothing here is live, can easily just keep iterating!

**Pull request merged**